src/EnergyPlus/Fans.cc

```diff
@@ -2643,8 +2643,7 @@ void FanSystem::set_size(EnergyPlusData &state)
             }
         }
     }
-    Real64 _rhoAir = Psychrometrics::PsyRhoAirFnPbTdbW(state, state.dataLoopNodes->Node(inletNodeNum).Press, inletAirTemp, inletAirHumRat);
-    designPointFEI = report_fei(state, maxAirFlowRate, designElecPower, deltaPress, _rhoAir);
+    designPointFEI = report_fei(state, maxAirFlowRate, designElecPower, deltaPress);
```

Comment:
report_fei is called in 2 places during sizing, 1 for the component model and another place for all other fan types. During sizing there is no indication of inlet condition other than what is determined by sizing. And if sizing is not requested then there is no data to grab with calling report_fei. So I think this is a reasonable solution to random 0's found for temperature at the inlet of the fan when this function is called. In the Eng Ref the equation is provided right above the description so the user does have a way to adjust this value if needed.

Reply:
@rraustad could you clarify if you are asking for changes here or not? The _random 0's_ phrase has me curious...

Reply:
I was just talking this through. The random 0's for T or w at the fan inlet node is what was previously used to calculate rhoAir, now rhoAir uses a reference condition (i.e., non-zero T and w). This is good to go.

    Real64 constexpr tempAirFan = 21.0; // Standard fan inlet temperature in Celsius
    Real64 constexpr hrAirFan = 0.5;    // Standard fan inlet humidity ratio (50%)
    Real64 _wAirFan = Psychrometrics::PsyWFnTdbRhPb(state, tempAirFan, hrAirFan, state.dataEnvrn->StdBaroPress);
    Real64 _rhoAirFan = Psychrometrics::PsyRhoAirFnPbTdbW(state, state.dataEnvrn->StdBaroPress, tempAirFan, _wAirFan);